ci-operator/step-registry/upi/install/heterogeneous/powervs/upi-install-heterogeneous-powervs-commands.sh

```diff
@@ -0,0 +1,148 @@
+
+      PATH=${PATH}:/tmp
+      mkdir -p ${IBMCLOUD_HOME_FOLDER}
+      if [ -z "$(command -v ibmcloud)" ]; then
+        echo "ibmcloud CLI doesn't exist, installing"
+        curl -fsSL https://clis.cloud.ibm.com/install/linux | sh
+      fi
+
+      function ic() {
+        HOME=${IBMCLOUD_HOME_FOLDER} ibmcloud "$@"
+      }
+
+      # Check if jq,yq and openshift-install are installed
+      if [ -z "$(command -v yq)" ]; then
+      	echo "yq is not installed, proceed to installing yq"
+      	curl -L "https://github.com/mikefarah/yq/releases/download/v4.30.5/yq_linux_$(uname -m | sed 's/aarch64/arm64/;s/x86_64/amd64/')" \
+          -o /tmp/yq && chmod +x /tmp/yq
+      else
+        echo "yq is already installed"
+      fi
+
+      if [ -z "$(command -v jq)" ]; then
+        echo "jq is not installed, proceed to installing jq"
+        curl -L "https://github.com/jqlang/jq/releases/download/jq-1.6/jq-linux64" -o /tmp/jq && chmod +x /tmp/jq
+      else
+        echo "jq is already installed"
+      fi
+
+      if [ -z "$(command -v openshift-install)" ]; then
+        echo "openshift-install is not installed, proceed to installing openshift-install"
+        curl -L https://mirror.openshift.com/pub/openshift-v4/multi/clients/ocp/stable/ppc64le/openshift-install-linux.tar.gz -o /tmp/openshift-install && chmod +x /tmp/openshift-install
+      else
+        echo "openshift-install is already installed"
+      fi
+
+      ic version
+      ic login --apikey @${CLUSTER_PROFILE_DIR}/ibmcloud-api-key -r ${REGION}
+      ic plugin install -f cloud-internet-services vpc-infrastructure cloud-object-storage power-iaas
+
+      # create workspace for power from cli
+      echo "Display all the variable values"
+      echo "Region is ${REGION} Resource Group is ${RESOURCE_GROUP}"
+      SERVICE_INSTANCE_OUTPUT=$(ic resource service-instance-create "${WORKSPACE_NAME}" "${SERVICE_NAME}" "${SERVICE_PLAN_NAME}" "${REGION}" -g "${RESOURCE_GROUP}")
+
+      SERVICE_INSTANCE_ID=$(echo "$SERVICE_INSTANCE_OUTPUT" | grep -oE 'GUID:[[:space:]]+[^:[:space:]]+' | awk '{print $2}')
+
+      echo ${SERVICE_INSTANCE_ID}
+      # After the workspace is created, invoke the automation code
+      cd ${IBMCLOUD_HOME_FOLDER} && git clone -b release-${OCP_VERSION} https://github.com/IBM/ocp4-upi-compute-powervs.git
+
+      # Set the values to be used for generating var.tfvars
+      IC_API_KEY="$(< "${CLUSTER_PROFILE_DIR}/ibmcloud-api-key")"
+      export IC_API_KEY
+      export PRIVATE_KEY_FILE=${CLUSTER_PROFILE_DIR}/ssh-privatekey
+      export PUBLIC_KEY_FILE=${CLUSTER_PROFILE_DIR}/ssh-publickey
+      export POWERVS_SERVICE_INSTANCE_ID=${SERVICE_INSTANCE_ID}
+      export INSTALL_CONFIG_FILE=${SHARED_DIR}/install-config.yaml
+      export KUBECONFIG=${SHARED_DIR}/kubeconfig
+
+      # Invoke create_var_file.sh to generate var.tfvars file
+      cd ${IBMCLOUD_HOME_FOLDER}/ocp4-upi-compute-powervs/scripts && chmod a+x create-var-file.sh && ./create-var-file.sh
```

Comment:
Awesome - Thanks for integrating... I raised a PR on this branch to add the import-image script.